instance/migrations/0101_auto_20180418_1601.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2018-04-18 22:21
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django_extensions.db.fields
+import instance.models.mixins.database
+
+
+class Migration(migrations.Migration):
```

Comment:
@rocioar Might be good to give this migration a sensible name (i.e. `add_mongodb_replia_set_model`)